internal/core/item_tree.rs

```diff
@@ -785,6 +785,31 @@ impl ItemRc {
     ) -> Option<R> {
         self.visit_descendants_impl(&mut visitor)
     }
+
+    /// Returns the transform to apply to children to map them into the local coordinate space of this item.
+    /// Typically this is the identity, but rotation for example may change this.
+    #[cfg(not(slint_int_coord))]
+    pub fn children_transform(&self) -> ItemTransform {
```

Comment:
I could also start right away using this in the rendering phase, and thus make the render() fn of the Rotate element empty. Thoughts?

Reply:
Indeed. Then we could even remove the `rotate` function from the ItemRenderer trait.
But one thing at the time is fine too.